scanpy/tests/notebooks/test_paga_paul15_subsampled.py

```diff
@@ -17,6 +18,7 @@
 FIGS = HERE / 'figures'
 
 
+@pytest.mark.skipif(not find_spec('louvain'), reason='needs package `louvain`')
 def test_paga_paul15_subsampled(image_comparer, plt):
     save_and_compare_images = image_comparer(ROOT, FIGS, tol=25)
```

Comment:
Why not `importorskip`?

```suggestion
def test_paga_paul15_subsampled(image_comparer, plt):
    pytest.importorskip("louvain")
    save_and_compare_images = image_comparer(ROOT, FIGS, tol=25)
```

Reply:
Good point! Hmm, too bad that one can’t be used as a mark/decorator, would be more consistent.